.prospector.yaml

```diff
@@ -0,0 +1,46 @@
+ignore-patterns:
+  - (^|/)\..+
+
+strictness: veryhigh
+
+pylint:
+  disable:
+  - fixme
+  - bad-continuation
+  - bad-indentation
+  - too-few-public-methods
+
+  options:
+    max-locals: 15
+    max-returns: 6
+    max-branches: 12
+    max-statements: 50
+    max-parents: 7
+    max-attributes: 7
+    min-public-methods: 2
```

Comment:
This will count inherited methods?

Reply:
not sure about that, but i assume it will.